cypress/support/elements/common/PrimaryWorkspace.js

```diff
@@ -84,9 +68,6 @@ class RightNav{
       return cy.get('.list.'+section+' [data-test='+section+'-list-items] .footer');
     }
```

Comment:
This function should also removed from here since it should be in ResourcePanel

Reply:
done